_release/8.0.0.md

```diff
@@ -0,0 +1,217 @@
+     9	Dominik Moritz
+     9	SHIMA Tatsuya
+     8	Neal Richardson
+     8	Sanjiban Sengupta
+     8	okadakk
+     8	Jacob Wujciak-Jens
+     7	Tobias Zagorni
+     7	Alessandro Molina
+     7	William Butler
+     6	Dhruv Vats
+     5	Yuqi Gu
+     5	Johnnathan
+     4	Anthony Louis
+     4	Projjal Chanda
+     4	Fiona La
+     4	Micah Kornfield
+     4	Bryan Cutler
+     3	Yaron Gvili
+     3	Ian Alexander Joiner
+     3	Vinicius Roque
+     3	ViniciusSouzaRoque
+     3	emkornfield
+     3	Chenxi Li
+     2	Jorge C. Leitao
+     2	Jose Almeida
+     2	assignUser
+     2	Uwe L. Korn
+     2	Todd Farmer
+     2	Tim Van Wassenhove
+     2	Rob Saccoccio
+     2	Alvin Chunga
+     2	Stephanie Hazlitt
+     2	Jeroen van Straten
+     2	Kyle Barron
+     2	Sasha Krassovsky
+     2	Eduardo Ponce
+     2	Salonijain27
+     2	liyafan82
+     2	JabariBooker
+     2	michalursa
+     2	Matthijs Brobbel
+     2	Danielle Navarro
+     2	Maya Anderson
+     2	Even Rouault
+     2	Joost Hoozemans
+     1	zhixingheyi-tian
+     1	SAm Albers
+     1	Alex McRae
+     1	Alfred Mountfield
+     1	Ali Amiri
+     1	Andrew Higgins
+     1	Andrew Strelsky
+     1	Andrii Oriekhov
+     1	Andy Teucher
+     1	Benjamin Kietzman
+     1	Benson Muite
+     1	Bob Matcuk
+     1	Chris Casola
+     1	Colin Jermain
+     1	Corentin
+     1	Dongjoon Hyun
+     1	Dooyoung-Hwang
+     1	Edward Visel
+     1	Elena Henderson
+     1	Frederic Branczyk
+     1	Haowei Yu
+     1	Hongze Zhang
+     1	Isaac Brodsky
+     1	James Duong
+     1	Johan Mattsson
+     1	João Pedro
+     1	Kazuaki Ishizaki
+     1	Kevin Gurney
+     1	Kian-Meng Ang
+     1	Laurent Goujon
+     1	Leo Di Donato
+     1	Li Jin
+     1	Min-Young Wu
+     1	MrMallIronmaker
+     1	Nate Clark
+     1	Nicolas Moreau
+     1	Nivia
+     1	Pradeep Garigipati
+     1	Rafael Telles
+     1	Romain Francois
+     1	Ryan Blue
+     1	Aakash Kumar
+     1	Sam Albers
+     1	Siddhant Rao
+     1	Stephan T. Lavavej
+     1	Thomas J. Fan
+     1	Tom Drabas
+     1	William Ayd
+     1	Xinyu Zeng
+     1	Yue Ni
+     1	Zac Farrell
+     1	bkmgit
+     1	burntcustard
+     1	fabiencelier
+     1	hankvyang
+     1	hzuo
+     1	iurysalino
+     1	jvictorhuguenin
+     1	jwijffels
+     1	karldw
+     1	marcbernot
+     1	okadak
+     1	ptaylor
+     1	shegden
+```
+
+## Patch Committers
+
+The following Apache committers merged contributed patches to the repository.
+
+```console
+$ git shortlog -csn apache-arrow-7.0.0..apache-arrow-8.0.0
+   105	David Li
+   102	Antoine Pitrou
+    84	Sutou Kouhei
+    72	Jonathan Keane
+    47	Krisztián Szűcs
+    46	Joris Van den Bossche
+    30	Weston Pace
+    25	Nic Crane
+    23	Pindikura Ravindra
+    22	Matthew Topol
+    16	Alessandro Molina
+    15	Yibo Cai
+    15	Neal Richardson
+    14	Dominik Moritz
+     5	Bryan Cutler
+     5	GitHub
+     3	Ian Cook
+     2	liyafan82
+     2	Micah Kornfield
+     1	Matt Topol
+     1	Li Jin
+     1	Chao Sun
+```
+
+## Changelog
+
```

Comment:
It seems the actual changelog is missing here? 
(eg compared to https://arrow.apache.org/release/7.0.0.html)

Reply:
Here is the PR https://github.com/apache/arrow-site/pull/210